core-drawer-panel.html

```diff
@@ -336,6 +336,11 @@
           }
           this.peeking = false;
         }
+
+        //disable selection while drawer is dragging
+        this.$.drawer.classList.add('noselect');
```

Comment:
Prefer declarative bindings rather than reaching into the DOM:

```html
<div id="main" class="{{ {noselect: dragging} | tokenList }}" ...>
```

```html
<div id="drawer" class="{{ {noselect: dragging} | tokenList }}" ...>
```